cssom-view-1/Overview.bs

```diff
@@ -1072,12 +1072,12 @@ dictionary ScrollIntoViewOptions : ScrollOptions {
 partial interface Element {
   DOMRectList getClientRects();
   [NewObject] DOMRect getBoundingClientRect();
-  void scrollIntoView(optional (boolean or ScrollIntoViewOptions) arg);
-  void scroll(optional ScrollToOptions options);
+  void scrollIntoView(optional (boolean or ScrollIntoViewOptions) arg = {});
```

Comment:
Are you sure this is the right thing given the `(boolean or ScrollIntoViewOptions)`?

Reply:
Yes. Previously the default value was implicitly an empty dictionary when the union includes non-required dictionary type, and now it must be explicit.

Reply:
Check out the relevant spec change: https://github.com/heycam/webidl/pull/750/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR2245